File: docs/mexc/mexc-api-sdk-main/mexc-api-sdk-main/examples/nodejs/changeMargin.js
import { MexcClient } from './MexcClient.js';

const client = new MexcClient({ apiKey: 'YOUR_API_KEY', isTestnet: true });

const result = await client.changeMargin({
  positionId: 1337,
  amount: 15,
  type: "ADD"
});

console.log(result);

File: docs/mexc/mexc-api-sdk-main/mexc-api-sdk-main/examples/python/change_margin.py
from mexc_client import MexcClient

client = MexcClient(api_key='YOUR_API_KEY', is_testnet=True)

result = client.change_margin({
    'positionId': 1337,
    'amount': 15,
    'type': "ADD"
})

print(result)
